plugins/org.jkiss.dbeaver.data.office/src/org/jkiss/dbeaver/data/office/export/WorksheetUtils.java

```diff
@@ -31,7 +31,7 @@ public class WorksheetUtils {
 
     private static final Log log = Log.getLog(WorksheetUtils.class);
 
-    public static final String DEFAULT_SHEET_NAME = "Sheet";
+    public static final String DEFAULT_SHEET_NAME = "CustomQuery";
```

Comment:
I believe this would be better
```suggestion
    public static final String DEFAULT_SHEET_NAME = "Custom query";
```